src/sentry/static/sentry/app/icons/iconActivity.tsx

```diff
@@ -1,23 +1,18 @@
 import React from 'react';
 
-import {IconProps} from 'app/types/iconProps';
-import theme from 'app/utils/theme';
+import SvgIcon from './svgIcon';
 
-export const IconActivity = React.forwardRef(function IconActivity(
-  {color: providedColor = 'currentColor', size: providedSize = 'sm', ...props}: IconProps,
-  ref: React.Ref<SVGSVGElement>
-) {
-  const color = providedColor;
-  const size = theme.iconSizes[providedSize] ?? providedSize;
+type Props = React.ComponentProps<typeof SvgIcon>;
 
-  return (
-    <svg viewBox="0 0 16 16" fill={color} height={size} width={size} {...props} ref={ref}>
-      <path d="M15.19,8.74H5.25a.75.75,0,0,1,0-1.5h9.94a.75.75,0,0,1,0,1.5Z" />
-      <path d="M15.19,15H5.25a.75.75,0,1,1,0-1.5h9.94a.75.75,0,0,1,0,1.5Z" />
-      <path d="M15.19,2.53H5.25a.75.75,0,0,1,0-1.5h9.94a.75.75,0,1,1,0,1.5Z" />
-      <path d="M2.25,8.74H.71a.75.75,0,1,1,0-1.5H2.25a.75.75,0,0,1,0,1.5Z" />
-      <path d="M2.25,15H.71a.75.75,0,0,1,0-1.5H2.25a.75.75,0,0,1,0,1.5Z" />
-      <path d="M2.25,2.53H.71A.75.75,0,0,1,.71,1H2.25a.75.75,0,1,1,0,1.5Z" />
-    </svg>
-  );
-});
+const IconActivity = (props: Props) => (
```

Comment:
Was the removal of the ref-forwarding intentional?

Reply:
yes...now I have it in the svgIcon 😉

Reply:
Does that work? Actually didn't think you could spread the ref prop.

Reply:
Hmm I have done it before and never had any issue so far. but just to be 100% sure let me do some research... good point